tsconfig.json

```diff
@@ -6,11 +6,11 @@
     ],
     "allowJs": true,
     "target": "ESNext",
-    "module": "ESNext", // Required for enabling JavaScript import assertion support
+    "module": "NodeNext", // Required for enabling JavaScript import assertion support
```

Comment:
Is there a particular reason why this was set to `NodeNext` when other packages are using `ESNext`:
[dwn-sdk-js](https://github.com/TBD54566975/dwn-sdk-js/blob/285036b386a14e354e8c3cebaca6d7d9aabcb1b6/tsconfig.json#L11)
[@web5/api](https://github.com/TBD54566975/web5-js/blob/8204a498e1abb07e4f3b85b416f989171379c141/packages/api/tsconfig.json#L10)

Reply:
Hm I've been experiencing an error, but let me investigate more on why...
<img width="921" alt="Screenshot 2023-08-31 at 2 26 01 PM" src="https://github.com/TBD54566975/dwn-server/assets/18248739/000f1d05-f61d-4f5a-bc99-dc350b3fbdd0">

Reply:
Oh, here is a reason: [introduced in typescript 5.2](https://devblogs.microsoft.com/typescript/announcing-typescript-5-2-rc/#module-and-moduleresolution-must-match-under-recent-node-js-settings)